### PR TITLE
Card v2

### PR DESCRIPTION
**Pull request by Ysrbolles**
# Description

Add new Component (card_v2) for New Website version.

## Type of change

- [x]  Add component

## Preview (if it’s a UI change):
Screen Shot 2022-08-25 at 10 47 06

# Checklist:

- [x]  My code follows the style guidelines of this project
- [x]  I have performed a self-review of my code
- [x]  I have commented on my code, particularly in hard-to-understand areas
- [x]  I have made corresponding changes to the documentation
- [x]  My changes generate no new warnings
- [x]  I have added tests that prove my fix is effective or that my feature works
- [x]  New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/Cards/Card_v2/index.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <CardContainer size={size}>
+      <CardImageContainer>
+        <Image
+          className="radius-xl"
+          role="img"
+          src={image}
+          width={size === 'small' ? 340 : 593}
+          height={380}
+          objectFit="cover"
+        />
```
Comment (redapy) [inline src/components/Cards/Card_v2/index.jsx:23]: please add an alt attribute

File: src/components/Cards/Card_v2/Card.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { screen, render } from '@testing-library/react'
+import { composeStories } from '@storybook/testing-react'
+import * as stories from './Card.stories'
+
+const { Small } = composeStories(stories)
+describe('Card', () => {
```
Comment (redapy) [inline src/components/Cards/Card_v2/Card.test.js:6]: Could you give a version of the card as well in the description? Just to not confuse it with the first Cards

File: src/components/Cards/Card_v2/Card.test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  it('should render card Default', async () => {
+    render(<Small />)
+    const card = await screen.findAllByRole('img')
+    expect(card).toHaveLength(1)
+  })
+  it('should render card with description', () => {
+    render(<Small />)
+    const card = screen.getByTestId('info')
```
Comment (redapy) [inline src/components/Cards/Card_v2/Card.test.js:14]: I believe using the text itself to get the element is better. we can use something like this `screen.getByText('Small.args.description')`.
Like this, we are testing that the props are also displayed correctly

**Review by redapy — CHANGES_REQUESTED** [review]
Great Work @Ysrbolles, there are just some small notes.

File: src/components/Cards/Card_v2/index.jsx (hunk 1 above)
Comment (Ysrbolles) [reply src/components/Cards/Card_v2/index.jsx]: Done, thank you for noticing me

File: src/components/Cards/Card_v2/Card.test.js (hunk 2 above)
Comment (Ysrbolles) [reply src/components/Cards/Card_v2/Card.test.js]: I's my bad, thank you

File: src/components/Cards/Card_v2/Card.test.js (hunk 3 above)
Comment (Ysrbolles) [reply src/components/Cards/Card_v2/Card.test.js]: For sure, It's fixed now, thank you

**Review by redapy — APPROVED** [review]

**Pull request merged**